src/components/Shared/TagCard.jsx

```diff
@@ -99,14 +99,53 @@ export default function TagCard(props) {
   return (
     <Card className={classes.card} raised>
       <CardContent className={classes.content}>
-        <Typography variant="body1" align="left" className={classes.tagHeading}>
-          Tag
-        </Typography>
+        <div style={{ display: 'flex', alignItems: 'center' }}>
+          <Typography variant="body1" align="left" className={classes.tagHeading}>
+            Tag
+          </Typography>
+          <IconButton
+            color="primary"
+            style={{ marginLeft: 'auto' }}
+            onClick={() => {
```

Comment:
In general, if your function is longer than 1 line, you want to declare it outside the JSX. Also, same thing abound using the window.confirm api